docs/root/intro/arch_overview/statistics.rst

```diff
@@ -4,23 +4,35 @@ Statistics
 ==========
 
 One of the primary goals of Envoy is to make the network understandable. Envoy emits a large number
-of statistics depending on how it is configured. Generally the statistics fall into two categories:
+of statistics depending on how it is configured. Generally the statistics fall into three categories:
 
 * **Downstream**: Downstream statistics relate to incoming connections/requests. They are emitted by
   listeners, the HTTP connection manager, the TCP proxy filter, etc.
 * **Upstream**: Upstream statistics relate to outgoing connections/requests. They are emitted by
   connection pools, the router filter, the TCP proxy filter, etc.
+* **Server**: Server statistics describes how the Envoy server instance is working. Statistics like
+  server uptime or amount of allocated memory are categorized here.
 
 A single proxy scenario typically involves both downstream and upstream statistics. The two types
 can be used to get a detailed picture of that particular network hop. Statistics from the entire
 mesh give a very detailed picture of each hop and overall network health. The statistics emitted are
 documented in detail in the operations guide.
 
-Envoy uses statsd as the statistics output format, though plugging in a different statistics sink
-would not be difficult. Both TCP and UDP statsd is supported. Internally, counters and gauges are
-batched and periodically flushed to improve performance. Histograms are written as they are
-received. Note: what were previously referred to as timers have become histograms as the only
-difference between the two representations was the units.
+In v1 API, Envoy only supports statsd as the statistics output format. Both TCP and UDP statsd
+are supported. Since Envoy has plugging mechanism to support different statistics sinks, as of v2
+API, multiple stats sinks are supported. Some of them emit tagged/multiple dimentions statistics.
```

Comment:
I really like the fact that you mention that whether tags are supported is sink-dependent - this is something that has caused issues for folks in the past.

I think it may be worth adding a separate paragraph/section on tags since they are user-configurable and not just an internal detail.  We should probably talk about the fact that Envoy stats have a canonical string representation that used throughout the documentation, and that the dynamic portions of the string are stripped from this name to produce tags and a static name within Envoy.  Then we can point to the portion of the v2 docs where the user can configure this behavior, where this is explained in more detail.

Also, nit: can we reword that sentence to "Some sinks also support emitting statistics with tags/dimensions."

Reply:
+1 to adding a dedicated scection about tagged statistics. How about this one? 56e6d48511d3089285fbf59997f6c08c41b01827